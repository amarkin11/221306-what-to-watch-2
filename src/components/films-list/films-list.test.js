
import React from "react";
import renderer from "react-test-renderer";
import FilmsList from "./films-list";

it(`Film card correctly renders after relaunch`, () => {
  const filmsListMock = [
    {
      name: `Macbeth`,
      src: `img/macbeth.jpg`
    },
    {
      name: `Aviator`,
      src: `img/aviator.jpg`
    },
  ];

  const tree = renderer
    .create(<FilmsList
      films={filmsListMock}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
