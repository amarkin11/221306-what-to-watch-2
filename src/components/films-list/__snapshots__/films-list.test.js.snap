// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Film card correctly renders after relaunch 1`] = `
Array [
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <video
      height="100%"
      muted={true}
      poster="img/macbeth.jpg"
      width="100%"
    >
      <source
        src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
        type="video/mp4"
      />
    </video>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Macbeth"
        height="175"
        src="img/macbeth.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Macbeth
      </a>
    </h3>
  </article>,
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <video
      height="100%"
      muted={true}
      poster="img/aviator.jpg"
      width="100%"
    >
      <source
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
        type="video/mp4"
      />
    </video>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Aviator"
        height="175"
        src="img/aviator.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Aviator
      </a>
    </h3>
  </article>,
]
`;
